tools/disasm/do_disasm.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+if [ "$VERBOSE" ]
+then
+echo "$VERSION"
+echo "$DISASM_DIR"
+echo "$DISASM_BASEROM"
+echo "$PYTHON"
+echo "$AS_CMD"
+echo "$LD"
+set -x
+fi
+
+DISASM_DATA_DIR="tools/disasm/$VERSION"
+DISASM_FLAGS="--custom-suffix _unknown --sequential-label-names --no-use-fpccsr --no-cop0-named-registers"
+DISASM_FLAGS="$DISASM_FLAGS --config-dir $DISASM_DATA_DIR --symbol-addrs $DISASM_DATA_DIR/functions.txt --symbol-addrs $DISASM_DATA_DIR/variables.txt"
+
+echo Disassembling...
+cmd="$PYTHON tools/disasm/disasm.py $DISASM_FLAGS $DISASM_BASEROM -o $DISASM_DIR --split-functions $DISASM_DIR/functions"
+echo "$cmd"
+$cmd || (
+    echo
+    echo Error on disassembling.
+    false
+)
+echo Disassembling done.
+
+echo Assembling sections individually...
```

Comment:
What would the individual sections be used for if we have the combined object now?

Reply:
I've discovered the individual section objects breaks `./retail_progress.py --csv` because it looks for .o files in expected (whether or not that's a good idea)

Reply:
The purpose would be building an OK rom by mix&match ing sections from build object and expected object

I will remove this until that's actually a thing, I guess (I'm working on it)